internal/controlplane/handlers_providers.go

```diff
@@ -47,7 +46,17 @@ func (s *Server) GetProvider(ctx context.Context, req *minderv1.GetProviderReque
 	})
 	if err != nil {
 		if errors.Is(err, sql.ErrNoRows) {
-			return nil, util.UserVisibleError(codes.NotFound, "provider not found")
+			// FIXME: This is temporary for backwards compatibility with old CLI versions
```

Comment:
The CLI previously queried the DB to get the authorization flows for the provider. This is to keep the server backwards compatible with older CLI versions.